common/src/chain/config/mod.rs

```diff
@@ -57,6 +57,7 @@ use super::{
 };
 
 const DEFAULT_MAX_FUTURE_BLOCK_TIME_OFFSET: Duration = Duration::from_secs(120);
+const MAX_FUTURE_BLOCK_TIME_OFFSET: Duration = Duration::from_secs(30);
```

Comment:
From `ChainConfig`'s point of view, both of them are "default". It's just that one of them is the old default and the other one is new.